src/main/java/com/iexec/blockchain/command/generic/CommandEngine.java

```diff
@@ -66,7 +66,7 @@ public String startBlockchainCommand(A args) {
                 chainObjectId, args);
 
         Runnable runnable = () -> triggerBlockchainCommand(args);
-        queueService.runAsync(runnable);
+        queueService.addExecutionToQueue(runnable, isPriority);
```

Comment:
I'd suggest something like `addHightPriorityExecutionToQueue(runnable)` and `addExecutionToQueue(runnable)`. The second one is neutral in terms of priority. It makes it more readable IMHO.

Reply:
Sure, but that implies a test in `startBlockchainCommand` that IMHO should not be there.